tests/sentry/issues/test_mark_reviewed.py

```diff
@@ -0,0 +1,38 @@
+from sentry.issues.mark_reviewed import mark_reviewed
+from sentry.models import GroupInbox, GroupInboxReason, add_group_to_inbox
+from sentry.testutils import TestCase
+
+
+class MarkReviewedTest(TestCase):  # type: ignore
+    def setUp(self) -> None:
+        super().setUp()
+        self.group = self.create_group()
+        self.group_list = [self.group]
+        self.group_ids = [self.group]
+        self.project_lookup = {self.group.project_id: self.group.project}
+        add_group_to_inbox(self.group, GroupInboxReason.NEW)
+
+    def test_mark_reviewed(self) -> None:
+        _ = mark_reviewed(
```

Comment:
```suggestion
        mark_reviewed(
```

Not sure there's any reason to capture the return value in the tests where you're not using it.